### ISSUE TITLE
Failure/Error: account.username

### ISSUE THREAD

**Issue by joanpla** [dhelonious/mastodon-snap#12]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

First, thank you so much for this! 

Now, I just tried it following the Quickstart guide, and everything run OK until the last step: 

`mastodon-server.tootctl accounts create administrator --email admin@example.com --role Owner --confirmed`

I get this error: 

`Failure/Error: account.username    
reserved`

I've tried different things, but with the same error output: 

- typing different emails (working and non-working) in "admin@example.com",
- sudo snap set mastodon-server email=myemail (different emails, including the one used )
- Adding the required MAIL info to /var/snap/mastodon-server/common/mastodon.conf

Also, since I want to use it just for one account, I set SINGLE_USER_MODE to true. 
BTW, just wondering: why do I need SMTP if I just want to use it for one account? Sorry, I guess this is a question for another place 😬.

PS: Not sure if it's relevant, but the url is https://mastodon.pla.cat 

### Version

I'm running it on a Hetzner VPS (CPX11)

### Architecture

amd64

### OS

Ubuntu 24.04

**Comment by dhelonious** [comment]
Sorry for the late reply, I completely missed the GitHub notification.

This is actually a problem in my documentation. The username `administrator` is reserved by Mastodon along with a few others. I thought this only affected the `admin` username. I have now fixed this, thanks for pointing this out!

Regarding your second question: Since the last spam wave that hit the Fediverse a few months ago, new accounts need to be approved. This seems to be the case for newly created owner accounts as well. As a workaround, you can use
```
mastodon-server.tootctl approve USERNAME
```
I have updated the readme in fe25e1c.

**Comment by joanpla** [comment]
Thank you so much!

Just a follow-up comment: Yes, it worked by changing the username and I received a random password. But when I logged in, I got lock on a screen with 2 contradictory messages (and couldn't go anywhere else):

- Your application is pending review by our staff. This may take some time. You will receive an e-mail if your application is approved.
- Your account is fully operational.

![Captura de pantalla 2024-10-08 212507](https://github.com/user-attachments/assets/4e6034c1-0d1f-4e00-881d-32be80f7ff79)

I found the solution in the official [documentation](https://docs.joinmastodon.org/admin/tootctl/#accounts-modify): adding '--approve', the user is approved ("--confirmed" is not enough). It's strange that the 'Owner' user needs to be approved...🤷
